Update all react-admin enterprise edition links in the documentation (#9772)
marmelab/react-admin: docs/Calendar.md

# `<Calendar>`

This [Enterprise Edition](https://react-admin-ee.marmelab.com)<img class="icon" src="./img/premium.svg" /> component, part of [the `ra-calendar` module](https://react-admin-ee.marmelab.com/modules/ra-calendar), renders a list of events as a calendar. 

<video controls autoplay playsinline muted loop>

>   <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-calendar.webm" type="video/webm" />

Reviewer: ```suggestion
  <source src="https://react-admin-ee.marmelab.com/assets/ra-calendar.webm" type="video/webm" />
```

>   <source src="https://react-admin-ee.marmelab.com/modules/assets/ra-calendar.mp4" type="video/mp4" />

Reviewer: ```suggestion
  <source src="https://react-admin-ee.marmelab.com/assets/ra-calendar.mp4" type="video/mp4" />
```

### `EditDialogProps`

> For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://react-admin-ee.marmelab.com/modules/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.

Reviewer: ```suggestion
For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog) to display its child form in a dialog.
```

> For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.

Reviewer: **issue**: does not redirect to the createdialog-editdialog section
```suggestion
For content edition, `<CompleteCalendar>` relies on [ra-form-layout's `<EditDialog>`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog) to display its child form in a dialog.
```

```
{% endraw %}

> Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://react-admin-ee.marmelab.com/modules/ra-form-layout#createdialog--editdialog).

Reviewer: ```suggestion
Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog).
```

> Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog--editdialog).

Reviewer: **issue**: same
```suggestion
Check the possible values for `EditDialogProps` in [the `<EditDialog>` component documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog).
```

### `CreateDialogProps`

> For content addition, `<CompleteCalendar>` relies on [ra-form-layout's `<CreateDialog>`](https://react-admin-ee.marmelab.com/modules/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.

Reviewer: ```suggestion
For content addition, `<CompleteCalendar>` relies on [ra-form-layout's `<CreateDialog>`](hhttps://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog) to display its child form in a dialog.
```

> For content addition, `<CompleteCalendar>` relies on [ra-form-layout's `<CreateDialog>`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog--editdialog) to display its child form in a dialog.

Reviewer: same
```suggestion
For content addition, `<CompleteCalendar>` relies on [ra-form-layout's `<CreateDialog>`](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog) to display its child form in a dialog.
```

```
{% endraw %}

> Check the possible values for `CreateDialogProps` in [the `<CreateDialog>` component documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog--editdialog).

Reviewer: same 
```suggestion
Check the possible values for `CreateDialogProps` in [the `<CreateDialog>` component documentation](https://react-admin-ee.marmelab.com/documentation/ra-form-layout#createdialog-editdialog--showdialog).
```
